README.md

```diff
@@ -1,65 +1,39 @@
-Bowling Challenge in Ruby
-=================
+# Bowling Challenge in Ruby
 
-* Feel free to use google, your notes, books, etc. but work on your own
-* If you refer to the solution of another coach or student, please put a link to that in your README
-* If you have a partial solution, **still check in a partial solution**
-* You must submit a pull request to this repo with your code by 9am Monday week
+This is a solution to the bowling challenge in Ruby. The challenge is to write a program that calculates the score of a bowling game.
 
-## The Task
+## How to use
 
-**THIS IS NOT A BOWLING GAME, IT IS A BOWLING SCORECARD PROGRAM. DO NOT GENERATE RANDOM ROLLS. THE USER INPUTS THE ROLLS.**
+Clone this repo and run bundle install to install the dependencies.
 
-Count and sum the scores of a bowling game for one player. For this challenge, you do _not_ need to build a web app with a UI, instead, just focus on the logic for bowling (you also don't need a database). Next end-of-unit challenge, you will have the chance to translate the logic to Javascript and build a user interface.
+To run the tests, run rspec from the command line.
```

Comment:
Currently the test on scorecard_spec.rb line 38 fails on rspec, but I am assuming that is because it is still in WIP

Reply:
Correct WIP